aesara/scalar/math.py

```diff
@@ -250,6 +250,39 @@ def c_code(self, node, name, inp, out, sub):
 erfcinv = Erfcinv(upgrade_to_float_no_complex, name="erfcinv")
 
 
+class Owens_t(BinaryScalarOp):
+    """
+    Implements the Owen's T function.
+    """
```

Comment:
```suggestion
```
This docstring isn't adding any extra information, so it's not necessary.  Also, I don't believe it's in NumPy format (i.e. it shouldn't start with a newline).